modules/tinymce/src/themes/silver/main/ts/Render.ts

```diff
@@ -165,6 +165,9 @@ const setup = (editor: Editor): RenderInfo => {
       onEscape: () => {
         editor.focus();
       },
+      onToolbarToggled: (state: boolean) => {
+        editor.dispatch('ToggleToolbarDrawer', { isToolbarDrawerToggled: state });
```

Comment:
This should be in the `api/Events.ts` file instead per the TinyMCE codestyle guidelines. The idea is to keep all the events fired by a plugin, theme, etc... to know what is used/exposed. See https://github.com/tinymce/tinymce/blob/ea5387c05c4a47a0aecb7289836a9c885ca6adf6/modules/tinymce/src/themes/silver/main/ts/api/Events.ts

Note: This is normally a lint rule but atm is disabled in TinyMCE due to some legacy things needing to be fixed.

- [ ] Move dispatch call to `api/Events.ts`